results/intfloat__multilingual-e5-small/LEMBNeedleRetrieval.json

```diff
@@ -0,0 +1,43 @@
+{
+  "dataset_revision": "6e346642246bfb4928c560ee08640dc84d074e8c",
+  "mteb_dataset_name": "LEMBNeedleRetrieval",
+  "mteb_version": "1.6.11",
+  "test": {
+    "evaluation_time": 1.76,
+    "map_at_1": 0.8,
+    "map_at_10": 0.87575,
+    "map_at_100": 0.87575,
+    "map_at_1000": 0.87575,
+    "map_at_20": 0.87575,
+    "map_at_3": 0.86667,
+    "map_at_5": 0.87067,
+    "mrr_at_1": 0.8,
+    "mrr_at_10": 0.87575,
+    "mrr_at_100": 0.87575,
+    "mrr_at_1000": 0.87575,
+    "mrr_at_20": 0.87575,
+    "mrr_at_3": 0.86667,
+    "mrr_at_5": 0.87067,
+    "ndcg_at_1": 0.8,
+    "ndcg_at_10": 0.90614,
```

Comment:
Seems like performance here is quite high. Does that correspond with your expectations? Not too familiar with the dataset, but it might be worth either using ndcg_at_1 or whatever they use in the paper.

Reply:
see below.